builtin/providers/aws/resource_aws_cloudtrail_test.go

```diff
@@ -193,6 +226,7 @@ resource "aws_cloudtrail" "foobar" {
     s3_bucket_name = "${aws_s3_bucket.foo.id}"
     s3_key_prefix = "/prefix"
     include_global_service_events = false
+    is_multi_region_trail = true
```

Comment:
Can you please omit this change in the data for the existing tests? It's breaking the existing two test cases:

```
=== RUN   TestAccAWSCloudTrail_basic
--- FAIL: TestAccAWSCloudTrail_basic (26.90s)
	testing.go:148: Step 1 error: Error applying: 1 error(s) occurred:

		* aws_cloudtrail.foobar: InvalidParameterCombinationException: Multi-Region trail must include global service events.
			status code: 400, request id: *REDACTED*
=== RUN   TestAccAWSCloudTrail_enable_logging
--- FAIL: TestAccAWSCloudTrail_enable_logging (27.56s)
	testing.go:148: Step 1 error: Error applying: 1 error(s) occurred:

		* aws_cloudtrail.foobar: InvalidParameterCombinationException: Multi-Region trail must include global service events.
			status code: 400, request id: *REDACTED*
```